src/lib/consts.js

```diff
@@ -60,6 +60,16 @@ exports.ACTS_TEMPLATES = {
         },
         skipOptionalDeps: true,
     },
+    apify_project: {
+        name: 'Apify project - Template for Apify projects, mainly for Apify Marketplace developers',
```

Comment:
What about: `Template containing boilerplate and code style rules used for Apify company internal projects.`

Reply:
or even better `Standardized template containing boilerplate and code style rules used for Apify Marketplace projects`

Reply:
This way marketplace developer will know they should use it